src/Modules.js

```diff
@@ -31,8 +31,8 @@ module.exports = {
   'invert': require('image-sequencer-invert'),
```

Comment:
The invert module comes from our own image-sequencer npm so the lower-case wont be reflected until new update, and I let it remain that way as this was different, using our repository through node_modules.